cdk/statsig/config.yaml

```diff
@@ -0,0 +1,21 @@
+message:
+  supportedMessageTypes:
+    - identify
+    - page
+    - screen
+    - alias
+    - track
+    - group
+  default:
+    transformation:
+      # Here no mapping is mentioned, so we'd check for mappings existence
+      # Since no mapping file(mapperPath) is found, we will simply send message here
+      # If spreadMessage is also not present, we should probably throw an error
```

Comment:
The comments here are not relevant. Please remove.

Reply:
Sure